benchmarks/nemo/nemolite2d/manual_versions/psykal_acc/time_step_mod.f90

```diff
@@ -427,14 +446,14 @@ subroutine invoke_time_step_arrays(istp, nx, ny, M, N,         &
 !$acc loop collapse(2)
     do jj = 1, N+1, 1
        do ji = 1, M, 1
-          call bc_solid_u_code(ji, jj, &
-                               ua, tmaskptr)
+          !call bc_solid_u_code(ji, jj, &
```

Comment:
With the function call I had a runtime error (pgi/19.10):
```
call to cuStreamSynchronize returned error 700: Illegal address during kernel execution
```
so I did switch to the inlined version without looking to much into it.

It is not clear to me why it fails because I see that the continuity is also not inlined and it works. Btw line 617 says the reason for not inlining this two functions was to check if the PGI compiler can cope with them. Since we now have Module-inline as a PSyclone transformation and we can check this easily with PSyclone, maybe we can delete this two functions and leave this file with all the kernels consistently inlined?

Reply:
Oh that's true, we can.

Reply:
Done, now all kernels are inlined.